static/app/components/events/traceEventDataSection.tsx

```diff
@@ -54,9 +60,8 @@ type Props = {
   recentFirst: boolean;
   stackTraceNotFound: boolean;
   stackType: STACK_TYPE;
-  title: React.ReactNode;
+  title: React.ReactElement<any, any>;
```

Comment:
did we change this because it can be `null` or `undefined`?

Reply:
I changed it because you cant cloneElement on null/undefined and it didnt look like null/undefined is intended usage.